moto/secretsmanager/models.py

```diff
@@ -501,15 +502,44 @@ def list_secret_version_ids(self, secret_id):
 
         return response
 
-    def list_secrets(self, filters, max_results, next_token):
-        # TODO implement pagination and limits
+    def list_secrets(self, filters: List, max_results: int = 100, next_token: str = None) -> Tuple[List, str]:
+        """
+        Returns secrets from secretsmanager.
+        The result is paginated and page items depends on the token value, because token contains start element
+        number of secret list.
+        Response example:
+        {
+            SecretList: [
+                {
+                    ARN: 'arn:aws:secretsmanager:us-east-1:1234567890:secret:test1-gEcah',
+                    Name: 'test1',
+                    ...
+                },
+                {
+                    ARN: 'arn:aws:secretsmanager:us-east-1:1234567890:secret:test2-KZwml',
+                    Name: 'test2',
+                    ...
+                }
+            ],
+            NextToken: '2'
+        }
 
+        :param filters: (List) Filter parameters.
+        :param max_results: (int) Max number of results per page.
+        :param next_token: (str) Page token.
+        :return: (Tuple[List,str]) Returns result list and next token.
+        """
         secret_list = []
         for secret in self.secrets.values():
             if _matches(secret, filters):
                 secret_list.append(secret.to_dict())
 
-        return secret_list, None
+        starting_point = int(next_token) if next_token else 0
+        ending_point = starting_point + int(max_results) if max_results else 100
```

Comment:
nitpick: I think `+` takes precedence over `if`/`else`, i.e., better write as:
```
ending_point = starting_point + (int(max_results) if max_results else 100)
```
or:
```
ending_point = starting_point + int(max_results or 100)
```

Reply:
Ah, sure, somehow missed that 😄 
Done.
Thanks @whummer!